### ISSUE TITLE
Meteor always determines that ios wkwebview is a legacy browser even though it is on par with safare

### ISSUE THREAD

**Issue by schlaegerz** [meteor/meteor#11612]
Ended up running into an issue where my code was running as the legacy bundle on ios, which was causing some issues.

Found out that I needed to call
```
setMinimumBrowserVersions(
    {
      "mobileSafariUI/WKWebView": 10,
    },
    "classes"
  );
```
To be able to mark it as a modern browser like it is.

This seems to be missing form the modern-browsers package either as an alias or as an additional item listed.

Seems like it would be a good default to include.

**Comment by StorytellerCZ** [comment]
For classes the minimum version of mobile Safari is set to 9.2, but the way you have added the name is not something that looks like we have listed in the package. Is this in Cordova?

**Comment by StorytellerCZ** [comment]
@schlaegerz can you please check that #11614 works for you?

**Comment by schlaegerz** [comment]
Yeah that looks like it works

**Comment by StorytellerCZ** [comment]
Published in `modern-browser@0.1.6`
